jenkins/bootstrap.py

```diff
@@ -992,7 +992,13 @@ def parse_args(arguments=None):
         '--clean',
         action='store_true',
         help='Clean the git repo before running tests.')
+    # split out args after `--` as job arguments
+    extra_job_args = []
+    if '--' in arguments:
```

Comment:
whoops - that's seems for unit-test only (patching up)

Reply:
https://github.com/kubernetes/test-infra/pull/4943